oracle_fdw.c

```diff
@@ -2480,41 +2544,17 @@ char
 	initStringInfo(&orderedquery);
 	appendStringInfo(&orderedquery, " ORDER BY");
 
-	/*
-	 * Determine whether we can potentially push query pathkeys to the remote
-	 * side, avoiding a local sort.
-	 */
-	foreach(cell, query_pathkeys)
-	{
+	/* If bestpath->pathkeys found, create sort_clause. */
```

Comment:
Thanks for the suggestion.
I'd like to divide into 2 processing such as check processing (determine pushdown or not)  and deparse processing. Therefore, that code was repeated.  But your suggestion is right. I'll fix it.